tools/run_tests/run_xds_tests.py

```diff
@@ -79,10 +88,11 @@
     'circuit_breaking',
     'timeout',
     'fault_injection',
+    'csds',
 ]
 
 # Test cases that require the V3 API.  Skipped in older runs.
-_V3_TEST_CASES = frozenset(['timeout', 'fault_injection'])
+_V3_TEST_CASES = frozenset(['timeout', 'fault_injection', 'csds'])
```

Comment:
CSDS doesn't require v3, does it?  Why does the test?

Reply:
The CSDS service definition itself needs to be at least v3. We as gRPC team has changed it a lot, so it has more features than v2.